docs/content/calico-cni.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Calico CNI"
+---
+
+When deploying a cluster with CAPI, deployment and configuration of CNI is up to the user. By leveraging CAPI cluster
+lifecycle hooks, this handler deploys Calico CNI on the new cluster via `ClusterResourceSets` at the
+`AfterControlPlaneInitialized` phase.
+
+Deployment of Calico is opt-in La cluster must be labelled with `capiext.labs.d2iq.io/cni=calico` for the lifecycle hook
```

Comment:
```
Deployment of Calico is opt-in where the cluster must be labelled ...
```

Reply:
To all my US friends, "labelled" is [not a typo](https://www.scribbr.com/us-vs-uk/labelled-or-labeled) :)